frontend/src/app/components/add-post-modal/add-post.component.ts

```diff
@@ -123,14 +123,32 @@ export class AddPostComponent {
       displayAttributes: null,
     };
 
-    return await this.canvasService.createBucketPost(boardID, post);
+    return await this.canvasService.createBucketPost(bucketID, post);
+  }
+
+  async addListPost() {
+    const post: Post = {
+      postID: generateUniqueID(),
+      userID: this.user.userID,
+      boardID: this.board.boardID,
+      author: this.user.username,
+      type: PostType.BUCKET,
```

Comment:
For list only posts, I think it'd be better to have their own type i.e. `PostType.LIST`.